openapi.yml

```diff
@@ -86,12 +86,30 @@ paths:
                 type: object
                 properties:
                   result:
-                    type: object
-                    properties:
-                      trace_id:
-                        type: integer
-                        description: ID of the trace executed
-                        example: 30001
+                    type: array 
+                    items:
+                      type: object
+                      properties:
+                        dpid:
+                          type: string
+                          description: Switch datapath ID 
+                          example: 00:00:00:00:00:00:00:01
+                        port:
+                          type: integer
+                          description: Incoming port in the switch
+                          example: 1
+                        time:
+                          type: string
+                          description: Start time in the first switch
+                          example: "2022-01-25 13:44:52.387021"
+                        type:
+                          type: string
+                          description: Type of the step. May be "starting", "trace", "last" or "loop".
```

Comment:
@gretelliz the current implementation doesn't support `last` or `loop` yet. It's great to have in the docs, but let's only add these when they're supported. Can you map a new issue to properly support `last` and `loop`? Since you'll work on major changes on flow_stats and sdntrace_cp I think you can add this minor fix there too. But let's continue priorizing the blueprint first, and then for now you remove `last` and `loop` from this PR, wdyt?

Reply:
You're right, I agree.

Reply:
Sounds good then @gretelliz, thanks for pushing the commit, looks good. I'll wait for the new issue to close this thread, if you need help to map this, let me know, maybe we can have a new epic for this `epic_sdntrace_cp_v1`, it'll be related to things that needs to be accomplished so feel free to take a look in the blueprint to decide on the new technical epics we'll go for (it's OK to have new technical epics as long as they're related to the original objectives of the version but we're just breaking them down).

Reply:
@viniarck I created issue #37

Reply:
Awesome @gretelliz. Thanks for mapping this and setting the `epic_sdntrace_cp_v1` label, let's close this thread then.